send/pithia_portal.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+import os
+import sys
+import subprocess
+import send_lib
+import requests
+
+SERVICE_NAME = "pithia_portal"
+os.environ["SERVICE_NAME"] = SERVICE_NAME
+
+facility = sys.argv[1]
+destination = sys.argv[2]
+destination_type = sys.argv[3]
+auth_token = None
+
+auth = send_lib.get_auth_credentials(SERVICE_NAME, destination)
+tokenUrl = send_lib.get_custom_config_properties(SERVICE_NAME, destination, ["tokenUrl"])[0]
+data = {
+	"username": auth[0],
+	"password": auth[1]
+}
+response = requests.post(tokenUrl, data)
+
```

Comment:
For some reason they need session cookie retrieved from the token endpoint so you can use this:
```python
resp_cookies = response.cookies.get_dict()
```